Docs/source/install/hpc.rst

```diff
@@ -45,3 +53,50 @@ HPC Systems
 
    Your HPC system is not in the list?
    `Open an issue <https://github.com/ECP-WarpX/WarpX/issues>`__ and together we can document it!
+
+
+.. _install-hpc-batch:
+
+Batch Systems
+-------------
+
+HPC systems use a scheduling ("batch") system for time sharing of computing resources.
+The batch system is used to request, queue, schedule and execute compute jobs asynchronously.
+The individual HPC machines above document job submission example scripts, as templates for your modifications.
+
+In this section, we document a quick reference guide (or cheat sheet) to interact in more detail with the various batch systems that you might encounter on different systems.
+
+Slurm
+"""""
+
+Slurm is a modern and very popular batch system.
+Slurm is used at NERSC, OLCF Frontier, among others.
+
+.. include:: batch/slurm.rst
+
+
+LSF
+"""
+
+LSF (for *Load Sharing Facility*) is an IBM batch system.
+It is used at OLCF Summit, LLNL Lassen, and other IBM systems.
+
+.. include:: batch/lsf.rst
+
+
+PBS
+"""
+
+PBS (for *Portable Batch System*) is a popular HPC batch system.
+The OpenPBS project is related to `PBS, PBS Pro and TORQUE <https://en.wikipedia.org/wiki/Portable_Batch_System>`__.
+
+.. include:: batch/pbs.rst
+
+
+PJM
+"""
+
+PJM (probably for *Parallel Job Manager*?) is a Fujitsu batch system
```

Comment:
@lucafedeli88 do you have further PJM references that we could use to check what PJM truly stands for in follow-up PRs?